tajo-client/src/main/java/org/apache/tajo/client/CatalogAdminClient.java

```diff
@@ -133,9 +135,10 @@ TableDesc createExternalTable(final String tableName, final Schema schema, final
    * @return Table description
    * @throws java.sql.SQLException
    */
-  TableDesc getTableDesc(final String tableName) throws SQLException;
+  TableDesc getTableDesc(final String tableName) throws UndefinedTableException;
 
-  List<CatalogProtos.FunctionDescProto> getFunctions(final String functionName) throws SQLException;
+  List<CatalogProtos.FunctionDescProto> getFunctions(final String functionName)
+      throws AmbiguousFunctionException, UndefinedFunctionException;
```

Comment:
It would be better if ```getFunctions``` returns an empty list rather than throwing an ```UndefinedFunctionException``` when there is no function of the given name. 
In addition, this method should return a list of the functions of the given name. So, ```AmbiguousFunctionException``` is inappropriate.